freva-rest/Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM alpine:latest as base
+FROM debian:bookworm-slim AS base
 ARG VERSION
 
 LABEL org.opencontainers.image.authors="DRKZ-CLINT"
 LABEL org.opencontainers.image.source="https://github.com/FREVA-CLINT/freva-nextgen/freva-rest"
 LABEL org.opencontainers.image.version="$VERSION"
 
-RUN apk update && \
-    apk add --no-cache \
-    python3 py3-pip python3-dev \
-    gcc g++ make curl openssl pkgconfig \
-    py3-setuptools py3-wheel build-base \
-    gnupg wget git \
-    libgcc libstdc++ \
-    openjdk11-jre bash procps
+# prerequisites
+RUN apt-get update && \
```

Comment:
don't you need the `DEBIAN_FRONTEND=noninteractive` env varialbe here?

Reply:
I will remove. But the main issue; if I want to bump version, then we need to change a bit in `freva-deployment` since this PR will effect on freva-rest-compose-template. What is your suggestion for this? only merge or bump version?